### ISSUE TITLE
TabsController - selecting tab item programmatically (issue)

### ISSUE THREAD

**Issue by melcug** [CosmicMind/Material#931]
In latest release of Material (v2.12.3) I can select a tab item programmatically, but the bar is not scrolled to make visible the selected tab button.

Code used to select a tab item by index:
```
tabsController?.select(at: tabIndex)
tabsController?.tabBar.select(at: tabIndex, completion: { (tabItem) in
	// ...
})
```

**Comment by danieldahan** [comment]
Taking a look now :)

**Comment by danieldahan** [comment]
You will find a fix in [Material 2.12.4](https://github.com/CosmicMind/Material/releases/tag/2.12.4) and you should only need to use: 

```swift
tabsController?.select(at: tabIndex)
```

Thank you and all the best!
